modules/openapi-generator/src/main/resources/go/model_simple.mustache

```diff
@@ -168,6 +168,18 @@ func (o *{{classname}}) Set{{name}}(v {{vendorExtensions.x-go-base-type}}) {
 {{/isNullable}}
 }
 
+{{^isReadOnly}}
+{{#defaultValue}}
+{{#initRequiredVars}}
+func (o *{{classname}}) GetDefault{{baseName}}() interface{}  {
```

Comment:
what about adding a docstring (a line or 2) explaining what this function does?

Reply:
I have addressed